scripts/checkout_ensembl.20210208.sh

```diff
@@ -96,25 +96,12 @@ done
 
 
 ## Now checkout Hive
-#branch="main"
-branch="version/2.6"
-for module in \
-    ensembl-hive
-do
-    echo "Checking out $module ($branch)" >> /dev/stderr
-    git clone -b $branch --depth 1 --no-single-branch ${URL_PFX}Ensembl/${module} || {
-        echo "Could not check out $module ($branch)" >> /dev/stderr
-        touch _FAILED
-        exit 2
-    }
-    [ -f _FAILED ] && exit 2
-    echo done >> /dev/stderr
-    echo
-done
 
 # slurm related bit
-mv ensembl-hive ensembl-hive.lsf
-git clone -b  feature/slurm_meadow --depth 1 --no-single-branch ${URL_PFX}Ensembl/ensembl-hive ensembl-hive.slurm 
+git clone -b  version/2.6 --depth 1 --no-single-branch ${URL_PFX}Ensembl/ensembl-hive ensembl-hive.lsf
```

Comment:
I would say since LSF is down from tomorrow, this line can be removed as well, right?